pkg/converter/tool/feature.go

```diff
@@ -28,11 +30,13 @@ const (
 	// stream into nydus blob directly, the tar2rafs eliminates the
 	// need to decompress it to a local directory first, thus greatly
 	// accelerating the pack process.
-	FeatureTar2Rafs Feature = "--type tar-rafs"
+	FeatureTar2Rafs  Feature = "--type tar-rafs"
+	FeatureBatchSize Feature = "--batch-size"
```

Comment:
The feature needs to be explained by a comment.

Reply:
Added.